cardano-binary/cardano-binary.cabal

```diff
@@ -16,7 +16,7 @@ build-type:          Simple
 extra-source-files:  README.md
                      CHANGELOG.md
 
-common base                         { build-depends: base >= 4.14 && < 4.19 }
+common base                         { build-depends: base >= 4.14 && < 5 }
```

Comment:
Is it a deliberate choice to move from a known-good upper bound on `base` to a "crystal ball" one? @lehins, opinion?

Reply:
Yes, other packages in this repo (eg `cardano-crypto-class`) already have a `base` upper bound of `5`.